tests/integration/integration_test.go

```diff
@@ -1452,3 +1452,25 @@ func TestPulumiNewLocalTemplatePath(t *testing.T) {
 	require.Empty(t, stderr)
 	require.Contains(t, stdout, "Your new project is ready to go")
 }
+
+func TestOverrideComponentNameAndNamespace(t *testing.T) {
+	t.Parallel()
+
+	e := ptesting.NewEnvironment(t)
+	defer e.DeleteIfNotFailed()
+
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+
+	e.ImportDirectory("component_test")
+	e.RunCommand("pulumi", "login", "--cloud-url", e.LocalURL())
+	e.CWD = filepath.Join(e.RootPath, "python-component")
+	e.RunCommand("python", "-m", "venv", "venv")
+	e.RunCommand(filepath.Join("venv", "bin", "pip"), "install", "-r", "requirements.txt")
+	e.RunCommand(filepath.Join("venv", "bin", "pip"), "install", "-e", filepath.Join(cwd, "..", "..", "sdk", "python"))
+
+	e.CWD = filepath.Join(e.RootPath, "ts-consumer")
+
+	stdout, _ := e.RunCommand("pulumi", "package", "add", "../python-component")
```

Comment:
Just set the `packages` section instead of the imperative command?

Reply:
I like to check the output of the command here since it includes the namespace.  Does `pulumi install` have that as well?

Reply:
So I just checked, `pulumi install` doesn't seem to do anything for local plugins (which maybe we should fix, but separately?).  So we need the `package add` here either way.

Reply:
Oh, it seems to work for me? Are you on the latest?

```yaml
cat Pulumi.yaml 
name: cert-component-yaml
runtime: yaml
packages:
  node-components: ../
resources:
  mycert:
    type: node-components:SelfSignedCertificate
    properties:
      dnsName: example.com
      validityPeriodHours: 24
      localValidityPeriodHours: 24
      subject:
        organization: Example Org
√ example % pulumi install
Installing packages defined in Pulumi.yaml...
Installing package 'node-components'...
Package 'node-components' installed successfully
```